tests/test_monitor.py

```diff
@@ -800,12 +800,12 @@ def test_filter_muxed_signal(self,
             [
                 call(0, 0, 'Received: 2, Discarded: 0, Errors: 0'),
                 call(1, 0, '   TIMESTAMP  MESSAGE                                           ', 'green'),
-                call(2, 0, '       0.000  BATTERY_VT('),
+                call(2, 0, '       0.000  BATTERY_VT :: 0('),
```

Comment:
hm, this is rather more unintuitive than I anticipated. do you think it would be better to include the selector signal name in the multiplexer name? something akin
```
BATTERY_VT :: BATTERY_VT_INDEX=0 (
 [...]
)
```

Reply:
Added multiplexor names